src/main/scala/blunt/Operator.scala

```diff
@@ -0,0 +1,32 @@
+package blunt
+
+import doobie.imports._
+import cats.implicits._
+
+sealed trait Operator
+
+case class lt() extends Operator 
+case class gt() extends Operator 
+case class gte() extends Operator
+case class lte() extends Operator
+case class eql() extends Operator
+
+trait ToSql[O <: Operator] {
+  def sql: Fragment
+}
+
+object Operator {
+  implicit val ltSql = new ToSql[lt] { val sql = fr"<" }
+  implicit val lteSql = new ToSql[lte] { val sql = fr"<=" }
+  implicit val gtSql = new ToSql[gt] { val sql = fr">" }
+  implicit val gteSql = new ToSql[gte] { val sql = fr">=" }
+  implicit val eqlSql = new ToSql[eql] { val sql = fr"=" }
+
+  object Symbols {
+    type < = lt
```

Comment:
@kentongray what do you think about this........merge this pr in as is and on Friday I will try and implement the like operator with cameron in the background.  This would be an effort to get more familiar with this repo so I can be more helpful in the code reviews?